#goal definition
assignment_1/final_pose pose

---
#result definition
assignment_1/obstacle_msg obstacle_pose
bool goal_reached
---
#feedback
float32 current_x
float32 current_y
float32 current_theta
string status